### PR TITLE
test(frontend): Improve v1 frontend-integration-test

### PR DESCRIPTION
**Pull request by jlyaoyuli**
Previously, we use `tab` key to switch between buttons and text fields. The main purpose of this PR is assign an ID to each input field and directly select it via the ID, which make the integration test more easily to maintain.

### CODE REVIEW

File: test/frontend-integration-test/helloworld.spec.js [hunk 1]
```diff
@@ -49,7 +54,9 @@ describe('deploy helloworld sample run', () => {
     await $('#localPackageBtn').click();
     const remoteFilePath = await browser.uploadFile('./helloworld.yaml');
     await $('#dropZone input[type="file"]').addValue(remoteFilePath);
-    await $('#newPipelineName').setValue(pipelineName);
```
Comment (jlyaoyuli) [inline test/frontend-integration-test/helloworld.spec.js:52]: Here is a hidden bug: setValue() won't clear default input value.

File: test/frontend-integration-test/helloworld.spec.js [hunk 2]
```diff
@@ -99,22 +106,14 @@ describe('deploy helloworld sample run', () => {
       reverse: true,
     });
 
-    await browser.keys('Tab');
+    await $('#runNameInput').click();
```
Comment (zijianjoy) [inline test/frontend-integration-test/helloworld.spec.js:109]: We should also `clearDefaultInput()` for the `#runNameInput` first, correct?

File: test/frontend-integration-test/helloworld.spec.js [hunk 3]
```diff
@@ -99,22 +106,14 @@ describe('deploy helloworld sample run', () => {
...
-    // Skip over service account help button
-    await browser.keys('Tab');
-    // Skip over "service account" textbox
-    await browser.keys('Tab');
-    // Skip over "Run Type" radio button
-    await browser.keys('Tab');
-
-    await browser.keys('Tab');
+    
+    await $('input#newRunPipelineParam0').click();
```
Comment (zijianjoy) [inline test/frontend-integration-test/helloworld.spec.js:115]: NIT: We can add a comment to explain what we expect this parameter name is for, for example: This is pipeline parameter `message`.

**Review by zijianjoy — COMMENTED** [review]
Good work Joe! There are two comments only.

**Review by zijianjoy — COMMENTED** [review]
/lgtm
/approve

Good work Joe!

**Pull request merged**